### PR TITLE
Bump chip wheels to 2024.4.1 (Matter SDK 1.3)

### PR DESCRIPTION
**Pull request by marcelveldt**
Bumps the wheels to version 2024.4 which is based on Matter SDK version 1.3

### CODE REVIEW

**Comment by marcelveldt** [comment]
This one is ready for review/merge.
Tested for several days and found no regressions/side effects.

**Review by agners — APPROVED** [review]

File: scripts/generate_devices.py
Comment (MartinHjelmare) [inline scripts/generate_devices.py:11]:
```suggestion
DEVICE_XML = (
    "https://raw.githubusercontent.com/project-chip/connectedhomeip"
    "/master/src/app/zap-templates/zcl/data-model/chip/matter-devices.xml"
)
```

**Review by MartinHjelmare — APPROVED** [review]

**Comment by marcelveldt** [comment]
not merge this one yet after we released a new stable version

**Pull request merged**
